google/cloud/spanner_dbapi/parse_utils.py

```diff
@@ -306,19 +306,15 @@ def parse_insert(insert_sql, params):
         # Case c)
 
         columns = [mi.strip(" `") for mi in match.group("columns").split(",")]
-        sql_params_list = []
-        insert_sql_preamble = "INSERT INTO %s (%s) VALUES %s" % (
-            match.group("table_name"),
-            match.group("columns"),
-            values.argv[0],
-        )
         values_pyformat = [str(arg) for arg in values.argv]
         rows_list = rows_for_insert_or_update(columns, params, values_pyformat)
-        insert_sql_preamble = sanitize_literals_for_upload(insert_sql_preamble)
-        for row in rows_list:
-            sql_params_list.append((insert_sql_preamble, row))
 
-        return {"sql_params_list": sql_params_list}
+        return {
+            "homogenous": True,
+            "table": match.group("table_name"),
+            "columns": columns,
+            "values": rows_list,
+        }
```

Comment:
Is this a breaking change?

Reply:
Nope i think.

Reply:
Oh, I see. The docstring correctly documents this behavior. In that case, I don't think this is a breaking change since it is fixing it to match the documented behavior.